.github/workflows/test-programs.yml

```
@@ -45,17 +45,26 @@ jobs:
 
+      - name: Set package name
+        id: program
+        run: |
+          if [ "${{ matrix.program }}" == "candy-guard" ]; then
+            printf 'package=%s\n' "mpl-core-candy-guard" >> "$GITHUB_OUTPUT"
+          else
+            printf 'package=%s\n' "mpl-core-candy-machine-core" >> "$GITHUB_OUTPUT"
+          fi
+
       - name: Run cargo fmt
         uses: actions-rs/cargo@v1
```

actionlint (1.7.12)

[error] the runner of "actions-rs/cargo@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)